File: PRAK405_2510817210004_MuhammadRizkyramadhani.c
#include <stdio.h>

int main() {
    int n, kelipatan, total = 0;

    printf("Selamat datang di program Jumlah Kelipatan Angka!\n");
    printf("_________________________________________\n\n");

    printf("Masukkan banyaknya n : ");
    scanf("%d", &n);
    printf("Masukkan nilai kelipatan : ");
    scanf("%d", &kelipatan);

    printf("_________________________________________\n\n");
    printf("Berikut adalah hasilnya :\n");

    for (int i = 1; i <= n; i++) {
        int jumlah_baris = 0;
        printf("");
        for (int j = i; j >= 1; j--) {
            jumlah_baris += j * kelipatan;
            printf("(%d * %d)", j, kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", jumlah_baris);
        total += jumlah_baris;
    }

    printf("Totalnya adalah %d\n", total);

    return 0;
}

File: PRAK405_2510817210004_MuhammadRizkyramadhani.py
# Program Jumlah Kelipatan
print('Selamat datang di program Jumlah Kelipatan Angka!')
print('_________________________________________\n')
n = int(input("Masukkan banyaknya n : "))
kelipatan = int(input("Masukkan nilai kelipatan : "))
print('_________________________________________\n')
print('Berikut adalah hasilnya :')

total = 0

for i in range(1, n + 1):
    jumlah_baris = 0
    ekspresi = ""
    for j in range(i, 0, -1):
        jumlah_baris += j * kelipatan
        ekspresi += f"({j} * {kelipatan})"
        if j > 1:
            ekspresi += " + "
    print(ekspresi, "=", jumlah_baris)
    total += jumlah_baris

print(f'Totalnya adalah {total}')